Issue 115 schema to single object

pengyin-shan opened the pull request:
This PR is for #115 and #121 . It will combine all JSONs in the `meta` folder into one JSON to use and add a 'Back to Result' button in the extended drawer.

### How to Test

1. Run the app and navigate to the search page
2. Perform searches and run filters, nothing should change in terms of functions
3. Open http://localhost:3000/api/metadata and you should see the combined JSON file being used
4. Open a drawer by clicking the 'More Info' button from one of the result
5. You should see a 'Back to Results' button in the drawer
6. Click the Back to Results' button. You should see the drawer closed

mradamcox reviewed (changes requested):
@pengyin, this is great, just a couple of things. 1) please change to /api/schema instead of /api/metadata, and 2) could you update your local indentation settings to match the existing settings? I noticed a couple of PRs ago that you have a different setting from the rest of the code base. If you think we should change our indentation permanently, we can do that all at once in a separate commit. It would be good to avoid these large diffs like you can see in this PR.

pengyin-shan commented:
> @pengyin, this is great, just a couple of things. 1) please change to /api/schema instead of /api/metadata, and 2) could you update your local indentation settings to match the existing settings? I noticed a couple of PRs ago that you have a different setting from the rest of the code base. If you think we should change our indentation permanently, we can do that all at once in a separate commit. It would be good to avoid these large diffs like you can see in this PR.

Sure, I updated the code 

> @pengyin, this is great, just a couple of things. 1) please change to /api/schema instead of /api/metadata, and 2) could you update your local indentation settings to match the existing settings? I noticed a couple of PRs ago that you have a different setting from the rest of the code base. If you think we should change our indentation permanently, we can do that all at once in a separate commit. It would be good to avoid these large diffs like you can see in this PR.

Hi @mradamcox , I added a `.editorconfig` for IDE to refer and will use this to avoid my listing error in the future. The new endpoint has been updated to `/api/schema`. Thank you!

mradamcox commented:
Great. Ready to merge this now.
